### PR TITLE
general-concepts/use-flags: hint towards CMAKE_DISABLE_FIND_PACKAGE_<pkg>

### PR DESCRIPTION
**Pull request by Flowdalic**
Motivated by https://github.com/gentoo/gentoo/pull/39625/files/9eb2f7bf5087777458cfc8838e43cdc533507d5b#r1874576245

### CODE REVIEW

**Comment by ulm** [comment]
Please wrap lines according to the [style guide](https://devmanual.gentoo.org/appendices/devbook-guide/index.html#internal-coding-style).

**Comment by Flowdalic** [comment]
> Please wrap lines according to the [style guide](https://devmanual.gentoo.org/appendices/devbook-guide/index.html#internal-coding-style).

Like this?

File: general-concepts/use-flags/text.xml [hunk 1]
```diff
@@ -32,9 +32,10 @@ Automagic dependencies are preferably fixed by preparing a build system patch
 adding appropriate options to control the dependency in question, and submitting
 this patch upstream for the benefit of all users. To avoid carrying additional
 patches downstream, automagic dependencies can usually be worked around using
-special build system options (e.g. cache variables in autotools) or through
-depending on the relevant packages unconditionally (i.e. forcing the check
-to always succeed).
+special build system options (e.g. cache variables in autotools or <c><uri
+link="https://cmake.org/cmake/help/latest/variable/CMAKE_DISABLE_FIND_PACKAGE_PackageName.html">
```
Comment (ulm) [inline general-concepts/use-flags/text.xml:36]: Almost:
```suggestion
special build system options (e.g. cache variables in autotools or
<c><uri link="https://cmake.org/cmake/help/latest/variable/CMAKE_DISABLE_FIND_PACKAGE_PackageName.html">
```

Comment (Flowdalic) [reply general-concepts/use-flags/text.xml]: k

**Comment by thesamesam** [comment]
Lgtm once formatting ok (can't check it on mobile)

**Review by ulm — APPROVED** [review]

**Comment by ulm** [comment]
Merged.

**Pull request closed without merging**